requirements.txt

```diff
@@ -3,3 +3,4 @@ pyaml
 pygal
 requests
 tqdm
+psycopg2
```

Comment:
Isn't `pyscopg2-binary` the package that should be used nowadays?

Reply:
Did you have any documentation that says to prefer `psycopg2-binary` over `psycopg2`. I can find only debate with unclear (to me) conclusion.